tests/generators/test_path.py

```diff
@@ -0,0 +1,53 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import retworkx
+
+
+class TestPathGraph(unittest.TestCase):
+
+    def test_directed_path_graph(self):
+        graph = retworkx.generators.directed_path_graph(20)
+        self.assertEqual(len(graph), 20)
+        self.assertEqual(len(graph.edges()), 19)
+        for i in range(19):
+            self.assertEqual(graph.out_edges(i), [(i, i + 1, None)])
```

Comment:
Check that there isn't one from 19 to 0?

Reply:
Oh, yeah that's a good call. I'll add that assert after the loop now

Reply:
fixed in: https://github.com/Qiskit/retworkx/pull/121/commits/bce9502291d5817ef44b081d75ec68bb6b4401a6